File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/statistikk/domene/SykefraværsstatistikkSektor.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.domene;

import lombok.AllArgsConstructor;
import lombok.Data;
import no.nav.arbeidsgiver.sykefravarsstatistikk.api.statistikk.domene.Sykefraværsstatistikk;

import java.math.BigDecimal;

@Data
@AllArgsConstructor
public class SykefraværsstatistikkSektor implements Sykefraværsstatistikk {
    private int årstall;
    private int kvartal;
    private String sektorkode;
    private int antallPersoner;

    private BigDecimal tapteDagsverk;
    private BigDecimal muligeDagsverk;
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/applikasjon/fellesdomene/SykefraværsstatistikkSektor.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.applikasjon.fellesdomene

import java.math.BigDecimal

data class SykefraværsstatistikkSektor(
    override val årstall: Int,
    override val kvartal: Int,
    val sektorkode: String,
    override val antallPersoner: Int,
    override val tapteDagsverk: BigDecimal?, // Må være nullable fordi klassen brukes som en kafkamelding ...
    override val muligeDagsverk: BigDecimal? // Må være nullable fordi klassen brukes som en kafkamelding ...
) : Sykefraværsstatistikk
